sources/osquery_thrift.js

```diff
@@ -9,6 +12,11 @@ const osqueryPlatforms = {
   linux: '../bin/osqueryi_linux'
 }
 
+const socketPath = {
+  darwin: `/Users/${process.env.USER}/.osquery`,
```

Comment:
Should/could this be in appData instead?

Reply:
It could and probably should, I went with using the default socket locations that `osquery` picks. I think I just need to run some tests to ensure that the provided socket path is reliably used.